bin/create-release-branch.js

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
-/* eslint-disable-next-line import/no-unassigned-import */
+/* eslint-disable-next-line import/no-unassigned-import,import/no-unresolved */
```

Comment:
I had to ignore `import/no-unresolved` here to enable running the linter in a parallel job to the build. It only passed the linter before because the build ran first.